src/Security/Security.php

```diff
@@ -1138,6 +1202,23 @@ public static function database_is_ready()
     }
 
     /**
+     * Resets the database_is_ready cache
+     */
+    public static function clear_database_is_ready()
+    {
+        self::$database_is_ready = null;
+        self::$force_database_is_ready = null;
```

Comment:
If it's a testing helper. Is it worth researching if it can be moved to the test itself? It's a static after all that isn't used anywhere _but_ in tests?